rules/0220-msauth_rules.xml

```diff
@@ -1160,6 +1160,16 @@
     <options>no_full_log</options>
   </rule>
 
+  <rule id="20021" level="3">
```

Comment:
This alert can make the alert 20019 to not appear when a user is logged physically for the first time in the system. I would change their order.